docs/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.0.x-to-4.1.8.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Migrate from 4.1.7 to 4.1.8 - Strapi Developer Docs
+description: Learn how you can migrate your Strapi application from 4.1.7 to 4.1.8.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.0.x-to-4.1.8.html
+---
+
+# v4.0.x to v4.1.8 migration guide
+
+The Strapi v4.0.x to v4.1.8 migration guide upgrades versions of v4.0.6 through v4.1.7 to v4.1.8. The minimum configuration for `config/admin` now includes the API token `API_TOKEN_SALT`. Additionally, Strapi no long populates default values for the admin JWT in `config/admin`. Initial values are generated and stored in the .env file during project creation. Strapi no longer passes secrets to non-developmen environments, requiring users to set the secrets purposefully. The migration to v4.1.8 consists of 3 steps:
```

Comment:
Simple "noob" question from a non-native English speaker:
Should it be "no long" or "no longer" (populates default values…)? 🤔

Reply:
(also I think we're missing a "t", "non-developmen" environments 😄

Reply:
Yes, it should be "longer". I think it was a copy and paste error or the delicate equilibrium of caffeine.